### DISCUSSION TITLE
Bus error on KE

### DISCUSSION THREAD

**Discussion by Tymec** [Guilouz/Creality-Helper-Script-Wiki#448] [Issues]
After installing Git Backup through the script on the KE, this error is spammed until I leave the SSH session. After reconnecting back the error seems to go away, but the plugin doesn't seem to work.
![image](https://github.com/Guilouz/Creality-K1-Series/assets/2727109/56cc221e-de11-4956-932a-1babed47f1b2)

I've also noticed this error shows up (without having Git Backup installed) when selecting the system menu in the script:
![image](https://github.com/Guilouz/Creality-K1-Series/assets/2727109/c5c295dd-eabb-4d92-a014-ff47a9e260c7)

Edit: After some troubleshooting I narrowed down the cause for the error when selecting the system menu, which was this line:
```sh
memfree=`cat /proc/meminfo | grep MemFree | awk {'print $2'}`
memtotal=`cat /proc/meminfo | grep MemTotal | awk {'print $2'}`
pourcent=$((($memfree * 100)/$memtotal))
```
For some reason `grep` throws a bus error
![image](https://github.com/Guilouz/Creality-K1-Series/assets/2727109/02e9330a-9dea-47e8-910d-30f0d86895f0)

**Comment by Guilouz** [comment]
You seem to have a problem with grep on your machine, can't reproduce this issue.

**Reply by Tymec** [reply]
I fixed it by restoring the firmware, but the issue with grep started occurring right after installing Git backup and persisted after removing it.

**Reply by Guilouz** [reply]
This happened because you didn't have the latest version of Entware.
